GUIs/ScriptRunnerPyQt5_GUI/scripFunc/scripWINONLY.py

```diff
@@ -524,13 +549,195 @@ def calladdusrtosys():
         calladdusrtosys()
 
     def remusr(self):
-        pass
+        class rmvusrfrosys(QDialog, Ui_rmvusrogru):
+            def __init__(self, parent = None):
+                super(rmvusrfrosys, self).__init__(parent)
+                self.setWindowIcon(QtGui.QIcon(':/Pictures/images/cup2.png'))
+                self.setupUi(self)
+                self.EXECUTE()
+
+            def EXECUTE(self):
+                self.setWindowTitle('Remove User From System')
+                self.label.setText('Current Users:')
+                self.label2.setText('Username: ')
+
+                def findnames():
+                    # Local users are added to a list of names
+                    EXEC = sub.Popen(["powershell", "& {net localgroup users}"], stdout = sub.PIPE)
```

Comment:
Codacy found an issue: [subprocess call - check for execution of untrusted input.](https://app.codacy.com/manual/vipersniper0501/CP_Scripts2/pullRequest?prid=5927419)